### PR TITLE
Add tooltips to flutter flame chart.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Screen Shot 2019-12-20 at 9 31 25 AM

This CL requires 0.1.8 publish of [flutter_widgets](https://pub.dev/packages/flutter_widgets) before landing.

### CODE REVIEW

File: packages/devtools_app/lib/src/charts/flutter/flame_chart.dart [hunk 1]
```diff
@@ -175,44 +182,67 @@ class _ScrollingFlameChartRowState extends State<ScrollingFlameChartRow>
...
-              child: node.buildWidget(node.data == widget.selected),
-            );
-          },
+    // Having each row handle gestures and mouse events instead of each node
+    // handling its own improves performance.
+    return MouseRegion(
+      onHover: _handleMouseHover,
+      child: GestureDetector(
+        behavior: HitTestBehavior.opaque,
+        onTapUp: (details) => _handleTapUp(details, context),
```
Comment (DaveShuckerow) [inline packages/devtools_app/lib/src/charts/flutter/flame_chart.dart]: Note: _handleTapUp can be a tearoff.  Context is an instance variable on state.

File: packages/devtools_app/lib/src/charts/flutter/flame_chart.dart [hunk 2]
```diff
@@ -175,44 +182,67 @@ class _ScrollingFlameChartRowState extends State<ScrollingFlameChartRow>
...
+          width: widget.width,
+          child: ListView.builder(
+            addAutomaticKeepAlives: false,
+            // The flame chart nodes are inexpensive to paint, so removing the
+            // repaint boundary improves efficiency.
+            addRepaintBoundaries: false,
+            controller: scrollController,
+            scrollDirection: Axis.horizontal,
+            itemCount: nodes.length,
+            itemBuilder: (context, index) {
```
Comment (DaveShuckerow) [inline packages/devtools_app/lib/src/charts/flutter/flame_chart.dart]: For brevity, I suggest splitting this out into a helper method.

File: packages/devtools_app/lib/src/charts/flutter/flame_chart.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/charts/flutter/flame_chart.dart]: done

File: packages/devtools_app/lib/src/charts/flutter/flame_chart.dart (hunk 2 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/charts/flutter/flame_chart.dart]: done

**Pull request merged**